rudder-server/SPECS/rudder-server.spec

```diff
@@ -234,6 +234,14 @@ if [ $1 -ne 1 ]
   fi
 
   getfacl --absolute-names --recursive /opt/rudder/etc/hooks.d/ > /tmp/rudder-hooks-upgrade
+
+  # Backup in case there was custom changes when it was a conffile
+  # not used anymore
```

Comment:
the equivalent of `case upgrade)` for rpm is` if [ $1 -ne 1 ]`

Reply:
it's already in a big `if [ $1 -ne 1 ]`

Reply:
aah sory